modules/islandora_oadoi/islandora_oadoi.install

```diff
@@ -13,3 +13,17 @@ function islandora_oadoi_uninstall() {
   );
   array_walk($vars, 'variable_del');
 }
+
+/**
+ * Implements hook_update().
+ *
+ * Deprecate and uninstall Islandora oaDOI module.
+ */
+function islandora_oadoi_update_7100() {
+  if (module_exists('islandora_oadoi')) {
```

Comment:
I don't think this is needed as for this code to be run the `islandora_oadoi` module would have to be installed.

Reply:
@bondjimbond but would this update hook get run if `islandora_oadoi` is not enabled? I can't see that it would, in which case I'm not sure this check is achieving anything.

Reply:
Oh wait, I see my mistake. Nevermind 🤦‍♂️